nucypher/config/constants.py

```diff
@@ -58,3 +58,4 @@
 
 # Web
 TEMPLATES_DIR = os.path.join(abspath(dirname(cli.__file__)), 'templates')
+MAX_UPLOAD_CONTENT_LENGTH = 1024 * 50
```

Comment:
How did you arrive at this number?

Reply:
@jMyles arrived at it after a brief discussion.